scripts/test-update-logs.go

```diff
@@ -1,3 +1,5 @@
+// +build skip
+
 package scripts
 
 import (
```

Comment:
does the lib on line 9 need to be a reference to the old go-ns /log library as well, in order to test that gets replaced?

Reply:
None of the apps should be using `go-ns/log` anymore because of the previous run of the script